packages/tasks/src/tasks/image-text-to-text/about.md

```diff
@@ -0,0 +1,75 @@
+## Different Types of Vision Language Models
+
+Vision language models come in three types:
+
+- **Base:** Pre-trained models that can be fine-tuned. A good example is for base models is [PaliGemma models](https://huggingface.co/models?sort=trending&search=google%2Fpaligemma-3b-pt) by Google.
+- **Instruction:** Base models fine-tuned on instruction datasets. A good example for instruction fine-tuned models is [idefics2-8b](https://huggingface.co/HuggingFaceM4/idefics2-8b).
+- **Chatty/Conversation:** Base models fine-tuned on conversation datasets. A good example for chatty models is [deepseek-vl-7b-chat](https://huggingface.co/deepseek-ai/deepseek-vl-7b-chat).
+
+![VLM uses](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/vlm/visual.jpg)
+
+## Use Cases
+
+### Multimodal Dialogue
+
+Vision language models can be used as multimodal assistants, keeping context about the conversation and keeping the image to have multiple-turn dialogues.
+
+### Zero-shot Object Detection, Image Segmentation and Localization
+
+Some vision language models can detect or segment a set of objects or describe the positions or relative positions of the objects. For example, one could prompt such a model to ask if one object is behind another. Such a model can also output bounding box coordination or segmentation masks directly in the text output, unlike the traditional models explicitly trained on only object detection or image segmentation.
+
+### Visual Question Answering
+
+Vision language models trained on image-text pairs can be used for visual question answering and generating captions for images.
+
+### Document Question Answering and Retrieval
+
+Documents often consist of different layouts, charts, tables, images and more. Vision language models trained on formatted documents can extract information from documents. This is an OCR-free approach, the inputs skip OCR and documents are directly fed to vision language models.
+
+### Image Recognition with Instructions
+
+Vision language models can recognize images through descriptions. When given detailed descriptions of specific entities, it can classify the entities in an image.
+
+## Inference
+
+You can use the Transformers library to interact with vision-language models. You can load the model like below.
+
+```python
+from transformers import LlavaNextProcessor, LlavaNextForConditionalGeneration
+import torch
+
+device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+processor = LlavaNextProcessor.from_pretrained("llava-hf/llava-v1.6-mistral-7b-hf")
+model = LlavaNextForConditionalGeneration.from_pretrained(
+    "llava-hf/llava-v1.6-mistral-7b-hf",
+    torch_dtype=torch.float16,
+    low_cpu_mem_usage=True
```

Comment:
I don't think we should put such a feature (somewhat experimental) in the task pages